src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -79,9 +79,10 @@ class IslandWindow :
     bool _alwaysOnTop{ false };
     bool _fullscreen{ false };
     bool _fWasMaximizedBeforeFullscreen{ false };
-    RECT _rcWindowBeforeFullscreen;
-    RECT _rcWorkBeforeFullscreen;
-    UINT _dpiBeforeFullscreen;
+    RECT _rcWindowBeforeFullscreen{};
+    RECT _rcWorkBeforeFullscreen{};
+    UINT _dpiBeforeFullscreen{ 1 };

```

Comment:
fwiw I don't think `1` is a sensible value here - `96` would be more appropriate, but leaving it as `1` would definitely do something obviously wrong if we ever tried using it before initializing